jetstream/core/metrics/prometheus.py

```diff
@@ -55,6 +55,60 @@ def __new__(cls):
       documentation="Total time taken to start the Jetstream server",
       labelnames=["id"],
   )
+  _request_input_length = Histogram(
+      name="jetstream_request_input_length",
+      documentation="Number of input tokens per request",
+      labelnames=["id"],
+      buckets=[
```

Comment:
I feel it's better to use same BUCKETS as prefill padding.

Reply:
+1

Reply:
Here is the code link: https://github.com/google/JetStream/blob/main/jetstream/engine/token_utils.py#L36

Reply:
Are we certain we want to differ from [the buckets vllm uses](https://docs.vllm.ai/en/latest/serving/metrics.html#:~:text=self.histogram_num_prompt_tokens_request%20%3D%20self._histogram_cls(%0A%20%20%20%20%20%20%20%20%20%20%20%20name%3D%22vllm%3Arequest_prompt_tokens%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20documentation%3D%22Number%20of%20prefill%20tokens%20processed.%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20labelnames%3Dlabelnames%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20buckets%3Dbuild_1_2_5_buckets(max_model_len)%2C%0A%20%20%20%20%20%20%20%20))?